uiowa/uiowa
# Sort Topic Collection Browse By Tag View nodes alphabetically

Resolves #5122

# How to test

1. Stay on the `main` branch
2. Sync the mentalhealth.uiowa.edu stage database into your local environment:
`blt sync --site mentalhealth.uiowa.edu --define drush.aliases.remote=mentalhealth.test`
3. Observe the ordering of the nodes under the "Browse by tag:" heading with the nodes in a bulleted list below the heading on this page: https://mentalhealth.uiowa.ddev.site/help
4. Check out this branch: `git checkout hotfix-topic-collection-browse-by-tag-sort`
5.  Run `drush @mentalhealth.local cim`
6. Observe the new ordering of the nodes in https://mentalhealth.uiowa.ddev.site/help, if they're now alphabetical, then this PR is successful
